src/utils.jl

```diff
@@ -6,9 +6,7 @@ const HEIGHT = Ref{Int}()
 const MODE = Ref{Symbol}(:default)
 
 function terminal_size(io)
-    ws = IOCTL.ioctl(io, IOCTL.TIOCGWINSZ)
-    # width, height
-    return (Int(ws.ws_col), Int(ws.ws_row))
+    return displaysize(stdout)
 end
 terminal_size() = terminal_size(stdout)
```

Comment:
I think it would be better if `terminal_size(io)` were left as is, and we changed `terminal_size()` to use `displaysize()`. Thoughts?

Reply:
You are correct, that is a better fix.